server/database/loginsettings.js

```diff
@@ -3,7 +3,7 @@
 //>CREATE ROLE working WITH LOGIN PASSWORD 'angel';
 //ALTER ROLE working CREATEDB;
 const settings = {
-  user: 'working',
+  user: 'robertbaker',
```

Comment:
I don't know if you need to put this file in .gitignore since it has your password...

Reply:
I have attempted to do this, but for some reason it appears the file is still visible and I'm not sure how to remove it from git tracking it since I already added it before. I have to also do this in the nosql branch so I will be investigating how to do it tonight, but I have taken your recommendation into consideration and added it to .gitignore.... it just apparently doesn't work if you've already committed the file once to just add it, you have to do something else.